fontbe/src/paths.rs

```diff
@@ -10,22 +10,29 @@ use crate::orchestration::WorkId;
 pub struct Paths {
     build_dir: PathBuf,
     glyph_dir: PathBuf,
+    debug_dir: PathBuf,
 }
 
 impl Paths {
     pub fn new(build_dir: &Path) -> Paths {
         let glyph_dir = build_dir.join("glyphs");
+        let debug_dir = build_dir.join("_debug");
```

Comment:
what's the meaning of the underscore prefix?

Reply:
I just felt it made it more obvious that it was different. I don't feel strongly about it, would just "debug" be better?

Reply:
i actually like the underscore prefix, it's like when you have an unused local variable to stop clippy from complaining, in this case _debug directory isn't actually "used" for compilation, hence the underscore